platform/security/cas/security-cas-client/src/main/java/org/codice/ddf/security/handler/cas/CasHandler.java

```diff
@@ -94,8 +96,10 @@ public HandlerResult getNormalizedToken(ServletRequest request, ServletResponse
                 handlerResult.setStatus(HandlerResult.Status.COMPLETED);
                 //update cache with new information
                 LOGGER.debug("Adding new CAS assertion for session {}",
-                        httpRequest.getSession(false)
-                                .getId());
+                        Hashing.sha256()
```

Comment:
❗️ For this one, since I assume the hash function is relatively expensive, you should wrap the `LOGGER.debug` statement in a check if `LOGGER.isDebugEnabled()`.